src/universal/modules/teamDashboard/components/ProviderRow/SlackProviderRow.tsx

```diff
@@ -20,21 +20,30 @@ import useMutationProps, {MenuMutationProps} from 'universal/hooks/useMutationPr
 import useAtmosphere from 'universal/hooks/useAtmosphere'
 import SlackClientManager from 'universal/utils/SlackClientManager'
 import SlackNotificationList from 'universal/modules/teamDashboard/components/ProviderRow/SlackNotificationList'
+import {DASH_SIDEBAR} from 'universal/components/Dashboard/DashSidebar'
+import useBreakpoint from 'universal/hooks/useBreakpoint'
 
 const StyledButton = styled(FlatButton)({
   borderColor: PALETTE.BORDER_LIGHT,
   color: PALETTE.TEXT_MAIN,
   fontSize: 14,
   fontWeight: 600,
+  minWidth: 36,
   paddingLeft: 0,
   paddingRight: 0,
   width: '100%'
 })
 
 const ProviderActions = styled(RowActions)({
   marginLeft: 'auto',
-  paddingLeft: Layout.ROW_GUTTER,
-  maxWidth: '10rem'
+  paddingLeft: 8,
```

Comment:
looks like this pattern is repeated 3 times. let's break it into its own component to share across the 3 integrations.
`RowActions` is pretty low value, we could probably just include all the props to avoid subclassing